tests/test_schema_evolution.py

```diff
@@ -48,7 +48,7 @@ def avro_to_bytes_with_schema(avro_schema, avro_dict):
 def bytes_with_schema_to_avro(avro_read_schema, binary):
     with BytesIO(binary) as bytes_io:
         reader = fastavro.reader(bytes_io, avro_read_schema)
-        return next(reader)
+        return reader.next()
```

Comment:
The iter_avro class in the master branch on this git repo does not define a __next__() function. It just defines Python 2 style next()